modules/axalloc/src/lib.rs

```diff
@@ -25,11 +25,11 @@ pub use page::GlobalPage;
 
 cfg_if::cfg_if! {
     if #[cfg(feature = "slab")] {
-        use allocator::SlabByteAllocator as DefaultByteAllocator;
+        pub use allocator::SlabByteAllocator as DefaultByteAllocator;
```

Comment:
Use the following code to make `rustdoc` works fine: 

```rust
cfg_if::cfg_if! {
    if #[cfg(feature = "slab")] {
        /// The default byte allocator.
        pub type DefaultByteAllocator = allocator::SlabByteAllocator;
    } else if #[cfg(feature = "buddy")] {
        /// The default byte allocator.
        pub type DefaultByteAllocator = allocator::BuddyByteAllocator;
    } else if #[cfg(feature = "tlsf")] {
        /// The default byte allocator.
        pub type DefaultByteAllocator = allocator::TlsfByteAllocator;
    }
}
```

Reply:
If remove os_dma，every driver lib needs glue code with `BusAddr` and `DMAInfo`, or you can add something like them in memory_addr crate.

Reply:
It has not been resolved.

Reply:
Use `type xxx = yyy` rather than `use xxx as yyy`